### PR TITLE
More on CA and reference domain ownership

### PR DESCRIPTION
**Pull request by fermulator**
**Description:**

I've been using the self-signed certificate option for some time, and was hunting for alternative options to transition to a trusted cert.

Going back to documentation, updating it with some context for those possibly unfamiliar (CA), and adding a reference to the domain ownership method.

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_docs/ecosystem/certificates/tls_self_signed_certificate.markdown [hunk 1]
```diff
@@ -6,7 +6,12 @@ redirect_from: /cookbook/tls_self_signed_certificate/
...
 [Let's encrypt]({{site_root}}/blog/2017/09/27/effortless-encryption-with-lets-encrypt-and-duckdns/) will only work if you have a DNS entry and remote access is allowed. 
-The solution is to use a self-signed certificate. As you most likely don't have a certification authority (CA) your browser will complain about the security. If you have a CA then this will not be an issue.
+
+The solution is to use a self-signed certificate. Please note however that after you have completed these steps, your browser will complain about the security of the certificate as it was not issued by a trusted authority.
+* This is due to self-signed certificates having not been issued by a certification authority ([`CA`](https://cheapsslsecurity.com/blog/what-is-a-certificate-authority-ca/)). If you have your own CA then this will not be an issue.
+* A fantastic workaround for this, while keeping your instance isolated securely off the Internet, is to use a [`Certificate for SSL/TLS via domain ownership`](/docs/ecosystem/certificates/tls_domain_certificate/).
```
Comment (fermulator) [inline source/_docs/ecosystem/certificates/tls_self_signed_certificate.markdown:13]: (please verify my markdown links are correct ;o)

File: source/_docs/ecosystem/certificates/tls_self_signed_certificate.markdown [hunk 2]
```diff
@@ -6,7 +6,12 @@ redirect_from: /cookbook/tls_self_signed_certificate/
 
 If your Home Assistant instance is only accessible from your local network you can still protect the communication between your browsers and the frontend with SSL/TLS. 
 [Let's encrypt]({{site_root}}/blog/2017/09/27/effortless-encryption-with-lets-encrypt-and-duckdns/) will only work if you have a DNS entry and remote access is allowed. 
-The solution is to use a self-signed certificate. As you most likely don't have a certification authority (CA) your browser will complain about the security. If you have a CA then this will not be an issue.
+
+The solution is to use a self-signed certificate. Please note however that after you have completed these steps, your browser will complain about the security of the certificate as it was not issued by a trusted authority.
+* This is due to self-signed certificates having not been issued by a certification authority ([`CA`](https://cheapsslsecurity.com/blog/what-is-a-certificate-authority-ca/)). If you have your own CA then this will not be an issue.
```
Comment (fermulator) [inline source/_docs/ecosystem/certificates/tls_self_signed_certificate.markdown]: there are MANY pages one could find/reference to explain what a CA is; I chose this one fairly arbitrarily, open to alternatives or preferences. (don't see what HA's policy is on acceptable external refs)

**Comment by fermulator** [comment]
Who can review?

File: source/_docs/ecosystem/certificates/tls_self_signed_certificate.markdown (hunk 2 above)
Comment (frenck) [reply source/_docs/ecosystem/certificates/tls_self_signed_certificate.markdown]: That looks fine, thanks!

File: source/_docs/ecosystem/certificates/tls_self_signed_certificate.markdown (hunk 1 above)
Comment (frenck) [reply source/_docs/ecosystem/certificates/tls_self_signed_certificate.markdown:13]: It is correct 👍

**Review by frenck — DISMISSED** [review]
Thanks, @fermulator! 👍

**Review by frenck — APPROVED** [review]

**Pull request merged**